app/components/form/multiple_select_component.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Form::MultipleSelectComponent < ViewComponent::Base
+  def initialize(form:, name:, options:, selected_items:, option_sub_text: false)
+    @form = form
+    @name = name
+    @options = options.to_json
+    @selected_items = selected_items
+    @option_sub_text = option_sub_text
```

Comment:
Nit: maybe `render_option_subtext`? I initially thought this would be the text itself (I think the text is in the options hash right?).

Reply:
I like it